toolchains/docker/toolchain.bzl

```diff
@@ -38,6 +40,10 @@ docker_toolchain = rule(
         "tool_path": attr.string(
             doc = "Path to the docker binary",
         ),
+        "client_config": attr.string(
+            default="",
```

Comment:
dont set default

Reply:
Keeping default on toolchain_rule as discussed offline because it will necessitate multiple BUILD.tpl files. Making the attribute on the repo rule not have a default